src/core/include/openvino/core/type/element_type.hpp

```diff
@@ -59,7 +60,8 @@ enum class Type_t {
     nf4,        //!< nf4 element type
     f8e4m3,     //!< f8e4m3 element type
     f8e5m2,     //!< f8e5m2 element type
```

Comment:
please, grep all other places where `f8e5m2` is used and add `f4e2m1` there.
E.g. C API, Java API in contrib

Reply:
Added for Java API in contrib will make separate PR.